mm2src/mm2_main/src/lp_swap/maker_swap.rs

```diff
@@ -745,7 +783,7 @@ impl MakerSwap {
         info!("Maker payment tx {:02x}", tx_hash);
 
         let tx_ident = TransactionIdentifier {
-            tx_hex: transaction.tx_hex().into(),
+            tx_hex: transaction.tx_hex().map(From::from),
```

Comment:
This is not critical at all, but my personal preference.
I prefer declaring the target type explicitly because it makes easier to read the code without jumping to the source code.
Also it `tx_hex` type is changed, we might want to change the conversion. For example, from `.map(BytesJson::from)` to `.map(|hex| hex.to_string().strip_prefix("0x"))`. In that case the hex will be converted with the prefix by default, and the engineer won't know about this.